api-server/src/server/boot/settings.js

```diff
@@ -291,3 +306,91 @@ function updateUserFlag(req, res, next) {
     message: 'flash.invalid-update-flag'
   });
 }
+
+function connectDiscourse(req, res) {
```

Comment:
The comments in this function seem redundant since the code is self-explanatory.